content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -34,170 +34,45 @@ To learn about the Kubernetes networking model, see [here](/docs/concepts/servic
-
-### OVN4NFV-K8s-Plugin (OVN based CNI controller & plugin)
-
-[OVN4NFV-K8S-Plugin](https://github.com/opnfv/ovn4nfv-k8s-plugin) is OVN based CNI controller plugin to provide cloud native based Service function chaining(SFC), Multiple OVN overlay networking, dynamic subnet creation, dynamic creation of virtual networks, VLAN Provider network, Direct provider network and pluggable with other Multi-network plugins, ideal for edge based cloud native workloads in Multi-cluster networking
-
-### NSX-T
-
-[VMware NSX-T](https://docs.vmware.com/en/VMware-NSX-T/index.html) is a network virtualization and security platform. NSX-T can provide network virtualization for a multi-cloud and multi-hypervisor environment and is focused on emerging application frameworks and architectures that have heterogeneous endpoints and technology stacks. In addition to vSphere hypervisors, these environments include other hypervisors such as KVM, containers, and bare metal.
-
-[NSX-T Container Plug-in (NCP)](https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) provides integration between NSX-T and container orchestrators such as Kubernetes, as well as integration between NSX-T and container-based CaaS/PaaS platforms such as Pivotal Container Service (PKS) and OpenShift.
-
-### OVN (Open Virtual Networking)
-
-OVN is an opensource network virtualization solution developed by the
-Open vSwitch community.  It lets one create logical switches, logical routers,
-stateful ACLs, load-balancers etc to build different virtual networking
-topologies.  The project has a specific Kubernetes plugin and documentation
-at [ovn-kubernetes](https://github.com/openvswitch/ovn-kubernetes).
-
-### Weave Net from Weaveworks
-
-[Weave Net](https://www.weave.works/oss/net/) is a
-resilient and simple to use network for Kubernetes and its hosted applications.
-Weave Net runs as a [CNI plug-in](https://www.weave.works/docs/net/latest/cni-plugin/)
-or stand-alone.  In either version, it doesn't require any configuration or extra code
-to run, and in both cases, the network provides one IP address per pod - as is standard for Kubernetes.
+* [Amazon AWS VPC CNI plugin](https://github.com/aws/amazon-vpc-cni-k8s)
+* [Amazon Calico for network policy enforcement](https://docs.aws.amazon.com/eks/latest/userguide/calico.html).
+* [Antrea](https://github.com/vmware-tanzu/antrea)
+* [Cilium](https://github.com/cilium/cilium) 
+* [Cisco Application Centric Infrastructure](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/application-centric-infrastructure/index.html)
```

Comment:
```suggestion
* [Cisco Application Centric Infrastructure(Cisco ACI)](https://www.cisco.com/c/en/us/solutions/data-center-virtualization/application-centric-infrastructure/index.html)
```
Is it better to include the word of `ACI`?

Reply:
Yes, good idea, added